presto-kafka/src/test/java/io/prestosql/plugin/kafka/KafkaQueryRunner.java

```diff
@@ -115,36 +97,14 @@ public Builder setExtension(Module extension)
         }
 
         @Override
-        public DistributedQueryRunner build()
+        public void preInit(DistributedQueryRunner queryRunner)
```

Comment:
Instead of these methods you can inject `Consumer<DistributedQueryRunner>` objects that could be called in certain points.

Reply:
I could inject a BiConsumer of KafkaQueryRunnerBuilder, DistributedQueryRunner, lmk.

Reply:
Let's leave it as is. `BiConsumer` is not much better.